app/invocation/invocation_action_card.tsx

```diff
@@ -241,23 +304,33 @@ export default class InvocationActionCardComponent extends React.Component<Props
         // proto field docs on `Execution.execute_response_digest`.
         const executeResponseBytes = actionResult.stdoutRaw;
         const executeResponse = build.bazel.remote.execution.v2.ExecuteResponse.decode(executeResponseBytes);
-        this.setState({ executeResponse });
-        if (executeResponse.result) {
-          const actionResult = executeResponse.result;
-          this.setState({ actionResult });
-          this.fetchStdout(actionResult);
-          this.fetchStderr(actionResult);
-          this.fetchServerLogs(executeResponse);
-        }
+        return executeResponse;
+      });
+  }
+
+  fetchExecuteResponseByActionDigest(actionDigest: build.bazel.remote.execution.v2.Digest) {
+    return rpc_service.service
```

Comment:
this should be using the "regional" rpc service via `getRegionalServiceOrDefault`, right?

Reply:
yup, done